### DISCUSSION TITLE
怎么样关闭右下角的 "checking for updates" ?

### DISCUSSION THREAD

**Discussion by transtone** [vuejs/language-tools#4885] [Q&A]
一直在那里转呀转的，既不能关闭，还不能隐藏，好烦人的。

看了代码，这里也不设置一个超时时间的吗，从早转到晚，从白天转到黑夜的。
https://github.com/vuejs/language-tools/blob/master/extensions/vscode/src/common.ts#L258

**Comment by transtone** [comment]
懒得看回复，自己编译了一个，给有同样困惑的小伙伴：https://github.com/transtone/language-tools/releases/tag/v2.1.6

**Comment by transtone** [comment]
update to v2.1.8:   https://github.com/transtone/language-tools/releases/download/v2.1.8/volar-2.1.8.vsix

**Comment by transtone** [comment]
update to v2.1.10:  https://github.com/transtone/language-tools/releases/download/v2.1.10/volar-2.1.10.vsix

**Comment by transtone** [comment]
update to v2.2.0: https://github.com/transtone/language-tools/releases/download/v2.2.0/volar-2.2.0.vsix

**Comment by KazariEX** [comment]
已经有超时机制了。
